frontend/src/slices/orders/orders.slice.ts

```diff
@@ -0,0 +1,38 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { DataStatus } from '~/libs/enums/enums.js';
+import { type ValueOf } from '~/libs/types/types.js';
+import { type OrderWithDriverEntity } from '~/packages/orders/libs/types/types.js';
+
+import { getOrder } from './actions.js';
+
+type State = {
+  order: OrderWithDriverEntity | null;
```

Comment:
Do we need a whole slice for one order? If we only need it on one page and there is always a new request on that page, maybe store that order in component state? I'm curious about the best way and would like to hear opinions @PavloSukhinin @kalinkaaaa14 @Druxight

Reply:
I think it also can be array of orders, now or in the future, advantage of having in it slice allows user get notifications when status updates during navigation on other pages